src/utils/shuffle-array.js

```diff
@@ -0,0 +1,10 @@
+export default function shuffleArray(array) {
```

Comment:
Let's'a add tests for this helper 🙏 

Also, I'd suggest reducing the utils modules by grouping a bit. We could do the classic group by data type and have a `array-utils.js` (I see there's a nice `string-utils.js` already).